index.html

```diff
@@ -3,10 +3,203 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="dist/style.css">
   <title>Cozy House</title>
 </head>
 <body>
-  
+  <section class="start">
+    <div class="containerHeaderFooter">
+      <div class="header">
+        <div class="logo">
+          <a href="#" title="Cozy House">Cozy House</a>
+          <p>Shelter for pets in Boston</p>
+        </div>
+        <div >
+          <ul class="nav-menu">
+            <li><a href="#" title="About the shelter" class="active">About the shelter</a></li>
+            <li><a href="#" title="Our pets" class="noactive">Our pets</a></li>
+            <li><a href="#" title="Help the shelter" class="noactive">Help the shelter</a></li>
+            <li><a href="#" title="Contacts" class="noactive">Contacts</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="startscreen">
+        <div class="textstart">
+          <h2>Not only people need a house</h2>
+          <p>We offer to give a chance to a little and nice puppy with an extremely wide and open heart. He or she will love you more than anybody else in the world, you will see!</p>
+          <div class="startbutton">
+            <a href="#" title="Make a friend">Make a friend</a>
+          </div>
+        </div>
+        <div><img src="img/start-screen-puppy.png" alt="puppy"></div>
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="container">
+      <div class="containerAbout">
```

Comment:
Do you think this `<div class="containerAbout">` is needed, what is a purpose of it?